web/src/components/Game/Runner/runner.js

```diff
@@ -0,0 +1,87 @@
+/* Physical body that models a runner */
+
+import Matter from 'matter-js';
+import RunnerSprite from './sprite';
+
+export default class Runner {
+  constructor(world) {
+    this.world = world;
+    this.sprite = new RunnerSprite(50);
```

Comment:
Yes - the starting position (400, 100) will need to be changed because in the "real" game, runners will probably be able to join after the game starts, at which point (400, 100) would be already off screen to the left.